TCAT/Controllers/AllStopsTableViewController.swift

```diff
@@ -185,4 +192,36 @@ class AllStopsTableViewController: UITableViewController, DZNEmptyDataSetSource,
         let attrs = [NSAttributedStringKey.foregroundColor: UIColor.mediumGrayColor]
         return NSAttributedString(string: title, attributes: attrs)
     }
+    
+    func buttonTitle(forEmptyDataSet scrollView: UIScrollView!, for state: UIControlState) -> NSAttributedString! {
+        let title = "Retry"
+        let attrs = [NSAttributedStringKey.foregroundColor: UIColor.buttonColor]
+        return NSAttributedString(string: title, attributes: attrs)
+    }
+    
+    func emptyDataSet(_ scrollView: UIScrollView!, didTap button: UIButton!) {
+        retryNetwork{() -> Void in
```

Comment:
make the code in the completion block a separate function so everything in the `emptyDataSet` function reads at the same level of abstraction